src/test/java/picard/fingerprint/FingerprintCheckerTest.java

```diff
@@ -375,7 +375,7 @@ void testCanHandleDeepData(final HaplotypeProbabilitiesFromSequence hp, final in
         final File fasta = new File(TEST_DATA_DIR, "reference.fasta");
```

Comment:
Do you want a test to show outputting multiple values?

Reply:
I'd love to, but it would involve building a bam...and I need to find time for that...

Reply:
when I find a project that actually uses this, I'll make time to write a test... OK?

Reply:
👍